worlds/atacama_y1a.world

```diff
@@ -51,8 +51,8 @@
             <exposure>0.05</exposure>
             <gamma>1.0</gamma>
             <energy_conversion>0.005</energy_conversion>
-            <read_noise>0.0</read_noise>
-            <shot_noise>0.0</shot_noise>
+            <read_noise_std_dev>0.0</read_noise_std_dev>
+            <shot_noise_coeff>0.0</shot_noise_coeff>
```

Comment:
Why do we initialize **read_noise_std_dev** and **shot_noise_coeff** to zeros? are these values realistic or we don't know their values for the camera in use?

Reply:
This is for the gzclient camera, so it doesn't need noise because it's not trying to simulate an actual camera. That said, the default noise params give so little noise you probably wouldn't notice it anyway.

The real reason we need to use the CameraSim plugin here is because we're rendering the scene with realistic light intensities far greater than 1.0, so the exposure and energy_conversion params are needed to remap pixel intensities to viewable values between 0 and 1.

Reply:
Makes sense .. so the stereo camera is just using defaults